ddtrace/internal/glob_matching.py

```diff
@@ -0,0 +1,50 @@
+from .utils.cache import cachedmethod
+
+
+class GlobMatcher:
```

Comment:
```suggestion
class GlobMatcher(object):
```

Python 2 compatibility, see https://stackoverflow.com/a/45062077